x-pack/plugins/apm/public/components/shared/aggregated_transactions_callout/index.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiCallOut } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+export function AggregatedTransactionsCallout() {
+  return (
+    <EuiCallOut
+      size="s"
+      title={i18n.translate('xpack.apm.aggregatedTransactions.callout.title', {
+        defaultMessage: 'No metrics events found',
+      })}
+      iconType="iInCircle"
+    >
+      <p>
+        {i18n.translate('xpack.apm.aggregatedTransactions.callout.content', {
+          defaultMessage: `This page is using transaction event data as no metrics events were found in the current time range.`,
```

Comment:
Let's replace the current `title` contents with this full description we can additionally decrease the size. This is also the pattern for the smaller callouts.

Reply:
I noticed @bmorelli25 already suggested the above in https://github.com/elastic/kibana/pull/108080#discussion_r686313801 - I'd prefer just only use the title for the small size.